scripts/commands/docs

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
```

Comment:
I think `#!/usr/bin/env bash` is the recommended scripts environment. but I don't mind that much.

Reply:
both are totally valid options. the one you have mentioned is more flexible when running the script with different versions and different user setups with different locations of the bash binary. The other one is meant to be more secure as users may have messed up their PATH env settings and unwanted bash binaries could be run